packages/public/frontend/package.json

```diff
@@ -12,6 +12,7 @@
     "lint": "yarn tsc --noemit",
     "serverless": "cross-env NEXT_ASSET_PREFIX=http://localhost:3000 ASSET_PREFIX=http://localhost:3000 node serverless/serverless.development.js",
     "start": "cross-env NEXT_ASSET_PREFIX=http://localhost:3000 ASSET_PREFIX=http://localhost:3000 next",
+    "start-dev": "cross-env NEXT_ASSET_PREFIX=http://localhost:3000 ASSET_PREFIX=http://localhost:3001 next",
```

Comment:
Why do we need to commands (`start` and `start-dev`) here?

Reply:
The start-dev command uses the frontend-assets service in docker setup, so its useful for me, maybe change the name to something like start-with-asset-server ...

Reply:
Mh... But isn't `next` already serving the assets in development anyways?

Reply:
sadly this leads to CORS issues